journal/history.go

```diff
@@ -0,0 +1,63 @@
+package journal
+
+import (
+	"io"
+	"os"
+	"path/filepath"
+	"sync"
+	"time"
+)
+
+var mux = sync.Mutex{}
+
+// Entry contains the command-line used to invoke the process associated with this item
+// and its journal ID.
+type Entry struct {
```

Comment:
Hmm, could we include this information in the filename + journal file? I'm thinking something like `<server_pid>-<process_pid>-<start_time>-<executable>` for the filename, and then the first line of the journal content could be `<description>`. The remaining content would be the activity.